README.md

```diff
@@ -33,12 +33,20 @@ Hello!  In the rush that is the start of the term, I really haven't had a chance
 
 Q1: So you're finishing up your time in this CSCC program.  Why did you decide to embark on this journey?
 
+* _Mark_: Originally started JAVA courses in 2014 
+
 Q2: I _love_ to travel.  Tell me a bit about your favorite travel destination so far.  What's on your travel bucket list?
 
+* _Mark_: St. John U.S. Virgin Islands was awesome.  I want to go to space.  Especially the Moon.  Dreamed of it my whole life.
```

Comment:
What was your St. John trip like?  Was it mostly a beach vacation or did you do other activities?  If it was centered around the beach, was there a particular beach you went to?  Please add a bit more detail to your response and update the PR.